src/js/profile/profile_loader.js

```diff
@@ -0,0 +1,42 @@
+import {Category} from "./category";
+
+const categoryClass = '.data-category';
+const profileHeaderClass = '#profile-top';
+const indicatorClass = '.indicator__sub-indicator';
```

Comment:
I'm not sure this is possible using Matt's setup, but it would be nice if this was anchored to a root element rather than global. Say all categories can be found in the `div id='#profile'>...</div>` element, then 

we have 

`export default class ProfileLoader {
    constructor(rootElement) {
        this.categoryTemplate = `${rootElement} ${categoryClass}`
       ...
   }
}
`

or even better

`export default class ProfileLoader {
    constructor(rootElement) {
        this.categoryTemplate = ${categoryClass, rootElement}
       ...
   }
}
`

what do you think?